variables.tf

```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
 
 variable "local_ip" {
   description = "URL used to find user's local IP for use with bastion host"
-  default     = "http://ipv4.icanhazip.com"
+  default     = "http://checkip.amazonaws.com"
   type        = string
 }
+
+variable "engine_version" {
+  description = "Aurora database engine version."
+  type        = string
+  default     = "10.7"
```

Comment:
@tonejito and @sjmiller609 What do you think about bumping this to the latest `11.6`? https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Updates.20180305.html

Reply:
I bumped it to `10.7` because our original instance was `10.6` (as the module specified) and it got updated to `10.7` on a maintenance window because `auto_minor_version_upgrade` was set to `true`

Don't know if hardcoding another version would force our Aurora RDS db to **create** a new instance or **migrate** the existing one :S

Reply:
I'm OK with 10.7 here

Reply:
I had 11.6 in my CI run and it failed. Let's stick with 10.7 for now and consider updating this in the future.